### ISSUE TITLE
ssh-add ./id_rsa.enc trouble

### ISSUE THREAD

**Issue by 0xm4ud** [nelsonic/hello-world-node-http-server#9]
Dear mr. Nelsonic, i`ve been trying to implement ssh, but i seem  have hit a wall, 
at that phase: ssh-add ./id_rsa.enc , i`m asked for a passphrase, what have i missed here, mate ?

thank you for time :)

**Comment by nelsonic** [comment]
@m4udSec are you trying to follow the instructions in https://github.com/dwyl/learn-devops/blob/master/nodejs-digital-ocean-centos-dokku.md ?

**Comment by 0xm4ud** [comment]
actually, i was reading an articles which makes references of you project.

**Comment by nelsonic** [comment]
@m4udSec so are you saying I'm _famous_...? ⭐️ `#finally`! 😜 

But seriously, please share a link to the article that references, I'm _really_ curious ... 🙏 

Hopefully the link above will help you trouble-shoot where you are getting stuck.
If not, please be a bit more specific with the step that is causing you a headache.
In what context are you running `ssh-add ./id_rsa.enc` ?
Are you attempting to automatically deploy from a CI? 💭 
If you're using Travis-CI (_the easiest one to work with_),
see: https://github.com/dwyl/learn-travis/blob/master/encrypted-ssh-keys-deployment.md

**Comment by 0xm4ud** [comment]
funny you`ve mentioned mate, that is one of the few materials that mentions you, check it out at the end of the page :), and yet , fame = power, and you know what uncle Ben says about powers :) yield it well my friend !

**Comment by 0xm4ud** [comment]
in the .travis.yaml
openssl aes-256-cbc -K $encrypted_77965d5bdd4d_key -iv $encrypted_77965d5bdd4d_iv -in deploy_key.enc -out ./deploy_key -d

the error: failed and exited with 1 during 
funny thing is : is if do remove this part it goes through, but ssh-add ./id_rsa.enc keep hangging, because its asks for a passphrase

**Comment by nelsonic** [comment]
@m4udSec in the steps I wrote I had to create an `ssh` key _without_ a password to get it to work.
(_I couldn't figure out how to do it with an ssh key that has a password ..._)

**Comment by 0xm4ud** [comment]
that .enc was derived from a key made without passphrase, that is what is bugging me !

**Comment by 0xm4ud** [comment]
do you have a special method used to create that key with no passphrase besides just hitting enter ?
by the way, thank you for helping me out, you are dealing very well with fame :)

**Comment by nelsonic** [comment]
Yeah, the way I created it was just hitting the `[Enter]` key to leave the password blank. 👍
